lpvartypes.pas

```diff
@@ -1827,7 +1843,18 @@ function TLapeType.Eval(Op: EOperator; var Dest: TResVar; Left, Right: TResVar;
         LapeExceptionFmt(lpeIncompatibleOperator1, [LapeOperatorToString(op), AsString])
       else
         LapeExceptionFmt(lpeIncompatibleOperator, [LapeOperatorToString(op)]);
-
+    except
+      on e: Exception do
+      begin
+        Dest := NullResVar;
+        Result := TryOperatorOverload();
```

Comment:
I think both `Eval` and `EvalRes` should be in the same place. Either both are done in `TLapeType` or in `TLapeTree_Operator`.